app/modules/ia_config_reader.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import json
+import os.path as path
+
+log = logging.getLogger(__name__)
+
+
+def load_config_to_dict(fname):
```

Comment:
Again this helper is only used in the constructor, it doesn't need to be a separate function